sass/base/_global.scss

```diff
@@ -0,0 +1,4 @@
+nav {
+    @include primary-font();
```

Comment:
Try to define the base font family (`$font-family-base`), instead of defining it only on this component.